runscripts/awiesm/PI_ctrl_awiesm-2.1-wiso_ollie.yaml

```diff
@@ -58,11 +43,13 @@ fesom:
     ini_parent_exp_id: piw_zstar_swon 
     ini_parent_dir: "/home/ollie/xshi/work/AWIESM_2_1_LR_concurrent_rad/piw_zstar_swon/restart/fesom/"
     ini_parent_date: "2681-12-31"
+
+    climate_data_dir: "/home/ollie/xshi/work/input/hydrography/"
```

Comment:
The variable climate_data_dir is already given to namelist_changes inside the fesom-2.0.yaml therefore you don't need to include it inside namelist_changes in your namelist.